documentation/docs/developers/connectors/index.md

```diff
@@ -10,18 +10,22 @@ There are [existing connectors](https://github.com/CoinAlpha/hummingbot/tree/mas
 
 Building a new exchange connector requires conforming to the template code to the new exchange's APIs, identifying and handling any differences in functions/behaviors, and testing the new exchange connector on that exchange.
 
-!!! Tip: "Recommendation"
-    For new developers building a connector, we recommend to use Binance as a template to start and understand the functions and behaviors.
 
-- Centralized exchange: 
-    - [Binance](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/binance) 
-    - [Coinbase Pro](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/coin-base_pro) - [Huobi](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/huobi)
-    - [Bittrex](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/bittrex)
-    - Ethereum DEX (0x open order book): [Radar Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/radar_relay)
-    - Ethereum DEX (0x open order book w/coordinator support): [Bamboo Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/bamboo_relay)
+!!! note "Recommendation"
+    For new developers building a connector, we recommend to use Crypto.com as a template to start understanding the functions and behaviors.
+
+The following list some examples/templates that you can refer to when building the connector:
+
+- [Crypto.com](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/crypto_com) 
+- [Binance](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/binance) 
+- [Coinbase Pro](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/coinbase_pro) 
+- [Huobi](hhttps://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/huobi)
+- [Bittrex](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/bittrex)
+- Ethereum DEX (0x open order book): [Radar Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/radar_relay)
+- Ethereum DEX (0x open order book w/coordinator support): [Bamboo Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/radar_relay)
 
 ## Exchange connector requirements
-1. A complete set of exchange connector files as listed [above](#exchange-connector-files).
+1. A set of exchange connector files as listed [above](#exchange-connector-files).
 2. Unit tests (see [existing unit tests](https://github.com/CoinAlpha/hummingbot/tree/master/test/integration)):
 
     1. Exchange market test ([example](https://github.com/CoinAlpha/hummingbot/blob/master/test/integration/test_binance_market.py))
```

Comment:
Can you change this to `test/connector/exchange/crypto_com/test_crypto_com_exchange.py`?

Reply:
edit the testing links to  crypto.com directory